sympy/printing/latex.py

```diff
@@ -846,6 +847,17 @@ def _print_ceiling(self, expr, exp=None):
         else:
             return tex
 
+    def _print_log(self, expr, exp=None):
+        if not self._settings["ln_notation"]:
+            tex = r"\log{\left (%s \right )}" % self._print(expr.args[0])
+        else:
+            tex = r"\ln{\left (%s \right )}" % self._print(expr.args[0])
+
+        if exp is not None:
+            return r"%s^{%s}" % (tex, exp)
```

Comment:
This is currently not tested. Can you please test this?

Reply:
you mean a test case when `exp` is not none?